paddle/platform/nccl_test.cu

```diff
@@ -33,7 +33,7 @@ TEST(NCCL, init) {
   comms.resize(dev_count);
   PADDLE_ENFORCE(dynload::ncclCommInitAll(comms.data(), dev_count, nullptr));
   for (int i = 0; i < dev_count; ++i) {
-    dynload::ncclCommDestroy(comms[i]);
+      dynload::ncclCommDestroy(comms[i]);
```

Comment:
Please indent 2 spaces as mentioned in https://google.github.io/styleguide/cppguide.html#Spaces_vs._Tabs

Reply:
this is just a left over from debugging. nothing should be changed here.